examples/testresample.rs

```diff
@@ -12,27 +12,30 @@ fn main() {
     let mut off = 0;
     let mut avail = INBLOCK as isize;
 
-    let fin : Vec<f32> = (0 .. INBLOCK * 2).map(|i| ((i as f32) / PERIOD * 2.0 * PI).sin() * 0.9).collect();
+    let fin: Vec<f32> = (0..INBLOCK * 2)
+        .map(|i| ((i as f32) / PERIOD * 2.0 * PI).sin() * 0.9)
+        .collect();
     let mut fout = vec![0f32; INBLOCK * 4];
 
     let mut st = State::new(1, RATE, RATE, 4).unwrap();
 
     st.set_rate(RATE, rate);
     st.skip_zeros();
 
-    st.set_quality(10).unwrap();
-    eprintln!("Quality: {}", st.get_quality());
```

Comment:
I think this `set_quality` is not important anymore, opinion?

Reply:
It is an example and setting quality is one of the few things you should care about when using the resampler. It should stay.